content/ideas/accessible-city-hackathon-app.html

```diff
@@ -0,0 +1,14 @@
+---
+title: Accessible City Hackathon Application: OCADU + Sidewalk Labs
```

Comment:
I think the ':' is causing issues when trying to generate the site. If you put quotes around the title, that should work.
title: "Accessible City Hackathon Application: OCADU + Sidewalk Labs"